src/lib/common/globals.cpp

```diff
@@ -188,6 +189,69 @@ void orionInit
 
 /* ****************************************************************************
 *
+* mongoInit -
+*/
+void mongoInit
```

Comment:
Why in common/globals.cpp?
Would it not be better to have the function in mongoBackend?
mongoBackend/mongoInit.h/cpp ...